runtime-light/core/globals/php-script-globals.cpp

```diff
@@ -0,0 +1,38 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2024 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include "php-script-globals.h"
+
+#include "runtime-light/component/component.h"
+
+PhpScriptMutableGlobals &PhpScriptMutableGlobals::current() noexcept {
+  return get_component_context()->php_script_mutable_globals_singleton;
+}
+
+void PhpScriptMutableGlobals::once_alloc_linear_mem(unsigned int n_bytes) {
+  php_assert(g_linear_mem == nullptr);
+  g_linear_mem = static_cast<char *>(get_platform_context()->allocator.alloc(n_bytes));
```

Comment:
Why not `alloc0`?

Reply:
platform allocator does not have alloc0

Reply:
Add `alloc0` to `RuntimeAllocator::global_memory_alloc`?